brouter-expressions/src/test/java/btools/expressions/IntegrityCheckProfileTest.java

```diff
@@ -0,0 +1,34 @@
+package btools.expressions;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+public class IntegrityCheckProfileTest {
+
+  @Test
+  public void integrityTestProfiles() throws IOException {
+    File workingDir = new File(".").getCanonicalFile();
+    File profileDir = new File(workingDir, "../misc/profiles2");
+    File[] files = profileDir.listFiles();
+
+    assertNotNull("Missing profiles", files);
+
+    for (File f : files) {
+      if (f.getName().endsWith(".brf")) {
+        BExpressionMetaData meta = new BExpressionMetaData();
+        BExpressionContext expctxWay = new BExpressionContextWay( meta );
+        BExpressionContext expctxNode = new BExpressionContextNode( meta );
+        meta.readMetaData( new File( profileDir, "lookups.dat") );
+        expctxNode.setForeignContext(expctxWay);
```

Comment:
Mixed style: Sometimes spaces after parenthesis and sometimes not. After the large reformatting we currently have a clean style within the repository and I'd like to keep it that way.

Reply:
> Is there any automatic for that?

If mean "Reformat code" and use IDEA, there are several shortcuts:
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html

Reply:
@devemux86 
Yes, that's the way it goes. But not very handy when you work with Notepad++.  So with automatic I mean 'reformat on save' or 'reformat on copy to git' or 'reformat an build with gradle' or whatever.

Reply:
AndroidStudio (which ist based on IntelliJ), VSCode and I guess some other IDEs offer formatting on save. If once setup you don't have to Care about formatting.

There ist no gradle task provided by checkstyle. You could add a git hook which checks formatting.